code/modules/mob/living/carbon/superior_animal/giant_spider/types/nurse.dm

```diff
@@ -105,6 +110,7 @@
 	egg_inject_chance = 15 //Defiently
 	//Giving the recluse its own special meat that has zombie powder. Reducing the amount of meat made since this is some hard stuff and the recluse is easy to kill.
 	poison_type = "zombiepowder"
+	armor_penetration = 70
```

Comment:
Actually okay with this one, these are meant to be scary and one hit is already an instant down.